src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const bodyDoc = document.querySelector('body');
+const mateLogo = document.querySelector('.logo');
+
+const promise1 = () => {
+  return new Promise((resolve, reject) => {
+    mateLogo.addEventListener('click', () => resolve('Promise was resolved!'));
+  });
+};
+
+const promise2 = () => {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+  });
+};
+
+promise1().then((val) => {
+  const div1 = document.createElement('div');
+
+  div1.setAttribute('class', 'message');
+  bodyDoc.appendChild(div1);
+
+  div1.textContent = val;
```

Comment:
Directly setting text content like this is fine for static strings, but if you're going to work with dynamic content, it's safer to use `textContent` to prevent potential cross-site scripting (XSS) attacks.